src/IoTeX/Staking.cpp

```diff
@@ -0,0 +1,217 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Staking.h"
+
+using namespace TW;
+
+namespace TW::IoTeX {
+
+/// Pack uint64 to uint256 in big-endian order
+///
+Data packUint64ToUint256(uint64_t value)
```

Comment:
can we resue methods in `uint256.h`? convert `uint64_t` to `uint256_t` then `encode256BE`